### PR TITLE
remove the unwanted code from webscoket sss-83

### PR DESCRIPTION
**Pull request by Arvind-Acc**
remove the unwanted code from webscoket

### CODE REVIEW

File: SmartApi/smartWebSocketV2.py [hunk 1]
```diff
@@ -186,27 +194,38 @@ def subscribe(self, correlation_id, mode, token_list):
...
+                    raise Exception(error_message)
+                else:
+                    self.wsapp.send(json.dumps(request_data))
+                    self.RESUBSCRIBE_FLAG = True
         except Exception as e:
-            print("Error:", e)
+            error_message = str(e)
+            if "Quota exceeded" in error_message:
+                logger.error(f"Quota exceeded error: {error_message}")
+                os._exit(1)  # Exit the script when the quota is exceeded
```
Comment (anurag-amx) [inline SmartApi/smartWebSocketV2.py:224]: this should be graceful handle.

Comment (anurag-amx) [inline SmartApi/smartWebSocketV2.py:224]: programmining does not allow this function until required

**Review by anurag-amx — CHANGES_REQUESTED** [review]

File: SmartApi/smartWebSocketV2.py (hunk 1 above)
Comment (Arvind-Acc) [reply SmartApi/smartWebSocketV2.py:224]: Remove the os._exit(1) and handle it gracefully

Comment (Arvind-Acc) [reply SmartApi/smartWebSocketV2.py:224]: remove the os._exit(1) and handle it gracefully

**Pull request merged**